content/docs/howto/php.md

```diff
@@ -27,147 +27,410 @@ for how to run the app.
 
-  # if included, will run `composer global` with specified arguments
-  install_global: ["list", "of", "install", "options"]
+{{< code/copyable >}}
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env BP_COMPOSER_INSTALL_GLOBAL="friendsofphp/php-cs-fixer squizlabs/php_codesniffer=*"
 {{< /code/copyable >}}
 
-## Select a Web Server
+### Set a custom Composer vendor directory
+To define a custom vendoring location for Composer packages, users can set the
+`COMPOSER_VENDOR_DIR` environment variable at build-time. It is a
+[Composer-native environment
+variable](https://getcomposer.org/doc/03-cli.md#composer-vendor-dir) setting.
+It must be set as a relative path under the application source directory root.
+
+{{< code/copyable >}}
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env COMPOSER_VENDOR_DIR="vendor"
+{{< /code/copyable >}}
+
+### Set the `composer.json` path
+To define a custom `composer.json` path, users can set [Composer-native
+environment variable
+`COMPOSER`](https://getcomposer.org/doc/03-cli.md#composer) at build-time. The
+path must be relative to the project root.
+
+{{< code/copyable >}}
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env COMPOSER="some-other-composer.json"
+{{< /code/copyable >}}
+
+### Set Composer authentication
+To set up authentication for Composer, the Composer-native environment variable
+`COMPOSER_AUTH` can be set as a JSON-formatted object containing objects
+defined in the [Composer
+docs](https://getcomposer.org/doc/03-cli.md#composer-auth).
+
+{{< code/copyable >}}
+pack build my-app --buildpack paketo-buildpacks/php \
+  --env COMPOSER_AUTH='{"http-basic": <some-creds>}'
+{{< /code/copyable >}}
+
+## Select a web server
 
 The PHP buildpack supports the use of 3 different web servers:
 
- - PHP built-in Web Server
- - Apache HTTP Web Server
+ - PHP Built-in Web Server (default)
+ - Apache HTTPD Web Server
  - Nginx Web Server
 
-You can configure the webserver using `buildpack.yml` as follows:
+You can configure which web server to use by setting the `BP_PHP_SERVER`
+environment variable at build-time.  The setting options are `php-server`,
+`httpd`, `nginx`, with the default value of `php-server` if unset. The PHP
+Built-in Server buildpack will only pass detection if there is a `*.php` file
+in the web directory of the application.
 
 {{< code/copyable >}}
-php:
-  # options are: php-server, httpd, nginx. default: php-server
-  webserver: php-server
+  pack build my-app --buildpack paketo-buildpacks/php \
+  --env BP_PHP_SERVER="php-server"
 {{< /code/copyable >}}
 
- If you're using `httpd` or `nginx`, a suitable `httpd.conf` or `nginx.conf`
- will be generated for you by the buildpack.
+#### Unsupported: Using buildpack.yml
+
+Specifying the PHP version through `buildpack.yml` configuration has been
+removed as of PHP language family buildpack v1.0.0.  To migrate from using
+`buildpack.yml` please set the environment variable equivalents instead.
```

Comment:
Also seems like copypasta from above. Maybe this is meant to be a different feature?